### PR TITLE
coala.py: Fixes "args.format" 

### LINKED ISSUE #3154: --format does not work in non-interative mode
Using `--ci` and `--format` together discards the --format argument.

### PR DESCRIPTION
**Pull request by shrutishrm512**
### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [x] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [x] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

Line 77 is modified by changing "args.format" to "args.format or args.ci"in coala/coalib/coala.py file to get same result for both the modes.

Closes #3154

### CODE REVIEW

**Comment by Nosferatul** [comment]
Please wrap the line from the commit body. Use `git commit --amend`, change the commit message and push force again!

**Comment by Mixih** [comment]
Tests failing.. marking WIP... Looks like you change introduced problems. Check the appveyor log for more info. (it doesn't look like there is an args.ci where do asked for one)
status/wip

**Comment by shrutishrm512** [comment]
--ci  and --format are two different modes, so as args.format exists for --format command then args.ci also need to exist for --ci command?

**Comment by arjunsinghy96** [comment]
Please do Fixes -> Fix and wrap the lines using `git commit --amend`. Please see http://docs.coala.io/en/stable/Getting_Involved/Writing_Good_Commits.html

**Comment by shrutishrm512** [comment]
@arjunsinghy96  Okay, i will amend the commit and then push it again!

File: coalib/coala.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def main():
     if args.non_interactive:
         return mode_non_interactive(console_printer, args)
 
-    if args.format:
+    if args.format or args.ci:
         return mode_format()
```
Comment (arjunsinghy96) [inline coalib/coala.py]: `--ci` and `--format` are not same. That is why adding args.ci on line 77 breaks stuff.
`--ci` and `--non-interactive` are same. Refer `coalib/parser/DefaultArgParser`. 
Make required changes.

Comment (arjunsinghy96) [inline coalib/coala.py]: run coala locally before pushing it.

**Comment by Mixih** [comment]
@arjunsinghy96 @shrutishrm512 
>Please do Fixes -> Fix and wrap the lines using git commit --amend. Please see http://docs.coala.io/en/stable/Getting_Involved/Writing_Good_Commits.html
Fixes is perfectly ok. Also, you linked to outed docs.

**Comment by arjunsinghy96** [comment]
@Mixih 
Wasn't updated about the new docs. Asked to do `Fixes -> Fix` due to the short log guidelines.

**Comment by Mixih** [comment]
Yeah, I misread sorry :(. And new docs are here https://api.coala.io/en/latest/
Carry on reviewing :)

**Comment by arjunsinghy96** [comment]
No Problem. Thanks for the doc link. :smile:

**Comment by shrutishrm512** [comment]
So, it's okay if we write `Fixes` :D

**Comment by Mixih** [comment]
@shrutishrm512 No it's not in the shortlog which is what he pointed out

**Comment by meetmangukiya** [comment]
@shrutishrm512 this does not solve the issue at all.. Even now either of the two modes will be executed, what we want in this issue is _both_ modes to work together.

**Comment by shrutishrm512** [comment]
`
    if args.non_interactive:
        return mode_non_interactive(console_printer, args)

    if args.format and args.non_interactive:
        return mode_format(console_printer, args)

    if args.format:
        return mode_format()
`
the changes should be like that? as it now 
coala --ci, coala --format and coala --format --non-interactive all three can work.

**Pull request closed without merging**